app/javascript/packages/document-capture/components/in-person-location-step.tsx

```diff
@@ -1,14 +1,190 @@
-import { PageHeading } from '@18f/identity-components';
-import { FormStepsButton } from '@18f/identity-form-steps';
+import { useState, useEffect, useCallback, useRef } from 'react';
 import { useI18n } from '@18f/identity-react-i18n';
+import {
+  PageHeading,
+  LocationCollectionItem,
+  LocationCollection,
+  SpinnerDots,
+} from '@18f/identity-components';
+
+interface PostOffice {
+  address: string;
+  city: string;
+  name: string;
+  phone: string;
+  saturday_hours: string;
+  state: string;
+  sunday_hours: string;
+  weekday_hours: string;
+  zip_code_4: string;
+  zip_code_5: string;
+}
+
+interface FormattedLocation {
+  formattedCityStateZip: string;
+  id: number;
+  name: string;
+  phone: string;
+  saturdayHours: string;
+  streetAddress: string;
+  sundayHours: string;
+  weekdayHours: string;
+}
+
+const locationUrl = '/verify/in_person/usps_locations';
+
+const getResponse = async () => {
+  const response = await fetch(locationUrl).then((res) =>
+    res.json().catch((error) => {
+      throw error;
+    }),
+  );
+  return response;
+};
```

Comment:
(Non-blocking) Is there any difference between explicitly catching and rethrowing, vs. not having the catch block at all?

```suggestion
const getResponse = () => fetch(locationUrl).then((res) => res.json());
```